### PR TITLE
request single robot

### PR DESCRIPTION
**Pull request by turbobeast**

### CODE REVIEW

File: src/Containers/App/App.js [hunk 1]
```diff
@@ -1,39 +1,22 @@
-import React, { Component } from "react";
+import React from "react";
```
Comment (turbobeast) [inline src/Containers/App/App.js]: use single quotes

File: src/actions.js [hunk 2]
```diff
@@ -2,23 +2,25 @@
...
   GET_ROBOTS_HAD_ERROR,
+  GET_ROBOT_HAD_ERROR,
 } from './constants';
 
 export const setSearchTerm = (term) => ({ 
   type: SET_SEARCH_TERM,
   payload: term,
 });
 
+const USER_API = "https://jsonplaceholder.typicode.com/users";
```
Comment (turbobeast) [inline src/actions.js]: use single quotes

**Review by turbobeast — COMMENTED** [review]
move App to components

**Comment by nathanielkess** [comment]
:shipit:

**Pull request merged**